app/src/main/java/com/bpn/trainme/presentation/features/home/HomeNavigation.kt

```diff
@@ -5,7 +5,6 @@ import androidx.navigation.compose.composable
 import com.bpn.trainme.domain.model.Exercise
 import kotlinx.serialization.Serializable
 
-
 @Serializable
```

Comment:
Since you are already having the kotlinx.serialization in the project, you can safely remove Gson and migrate the usage of Gson to Serialization Lib